server/core/src/https/ui.rs

```diff
@@ -83,11 +80,7 @@ pub(crate) async fn ui_handler_generic(
 
     let jstags = jsfiles.join("\n");
 
-    let body = format!(
-        include_str!("ui_html.html"),
-        domain_display_name.as_str(),
-        jstags,
-    );
+    let body = format!(include_str!("ui_html.html"), domain_display_name, jstags,);
```

Comment:
```suggestion
    let body = format!(include_str!("ui_html.html"), domain_info.display_name(), jstags);
```

Reply:
Something something the borrow checker might be cranky but it should be an easy fix 😄

Reply:
I'll try it :)